src/components/PokemonDisplay.js

```diff
@@ -0,0 +1,107 @@
+import React, {setState, useEffect} from 'react';
+import $ from 'jquery'
+import Button from '@material-ui/core/Button'
+
+class PokemonDisplay extends React.Component {
+
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            index: 0,
+            value: "",
+            pokemon: {
```

Comment:
Consider making a Pokemon interface / class. And move its properties and the html parsing logic to it

Reply:
Moved all Pokemon related information/methods to their own class.